src/components/Chat/BottomSheetGif.js

```diff
@@ -9,6 +9,7 @@ import {
   FlatList
 } from 'react-native';
 import FastImage from 'react-native-fast-image';
+import {debounce} from 'lodash';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `debounce` from lodash is a good addition as it helps to limit the rate at which a function can fire, in this case, the search function. This can help improve performance by reducing unnecessary API calls when the user is typing in the search box.
```diff
+ import {debounce} from 'lodash';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->